WebInstall/explainer_same_domain.md

```diff
@@ -72,7 +72,7 @@ When called on the same domain, the **`install` method will trigger/open the pro
 
 ### The `navigator.install` method
 
-To install a web site/app, the site/app would use the promise-based method`navigator.install(<id>[[, install_url], <params>])`. This method will:
+To install a web site/app, the site/app would use the promise-based method `navigator.install(id, install_url, [<params>])`. This method will:
 
 * Resolve when an installation was completed.
     * The success value will be an object that contains:
```

Comment:
Is the `id` "computed" at this point?  Based on this design, web install no longer permits the usage of computed ids, only explicitly defined id's declared in the manifest.

Reply:
Need to update 2. below ~100 to say "computed or id in manifest"